plugins/nessus.rb

```diff
@@ -348,7 +348,7 @@ def cmd_nessus_connect(*args)
         case args.length
         when 1,2
           if args[0].include? "@"
-            cred,targ = args[0].split('@', 2)
+            cred,doesnotmatter,targ = args[0].rpartition('@')
```

Comment:
A similar fix was required for the nexpose plugin over here :+1:

https://github.com/rapid7/metasploit-framework/pull/14962